### ISSUE TITLE
Add `temperature`, `top_k`and `logits`for local models

### ISSUE THREAD

**Issue by andthattoo** [andthattoo/ollama-workflows#18]
We should have the option to obtain logits set temperature and top_k for ollama models
